### PR TITLE
Advocacy site: updates to map & info box.

### PR DESCRIPTION
**Pull request by breville**
## advocacy.code.org

no state selected

screenshot 2018-04-11 17 14 18

wide page (narrow box)

screenshot 2018-04-11 17 14 31

narrow page (wide box)

screenshot 2018-04-11 17 14 51

## code.org/promote

no state selected

screenshot 2018-04-11 17 16 17

wide page (narrow box)

screenshot 2018-04-11 17 16 28

narrow page (wide box)

screenshot 2018-04-11 17 16 42

Also some content changes to the four quarter slots on http://advocacy.code.org

### CODE REVIEW

**Comment by tanyaparker** [comment]
Some thoughts

- What do you think about making the state's name in the box teal too? So we don't have green/yet another color? Then you could change the /promote highlight color to teal to keep the highlight and title color the same on /promote too.
- I don't think the `x` and `check` icons need to be so big. But we can try it. Why not.
- The orange buttons look so tall. Do you think they need that much padding?
- Feels like the 3 policies should be centered on the narrow page, wide box too.

**Comment by tanyaparker** [comment]
Can you include screenshot of the box before a state is selected?

File: pegasus/sites.v3/code.org/views/promote_state.haml [hunk 1]
```diff
@@ -137,9 +137,6 @@
     font-weight: bold;
     text-align:center;
   }
-  .fact {
-    font-size:15px;
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/views/promote_state.haml]: teeny tiny nit: space between font-size and 15px

Comment (breville) [reply pegasus/sites.v3/code.org/views/promote_state.haml]: Fortunately that is being removed :)

**Comment by breville** [comment]
- colours updated per offline discussion.
- :)
- reduced them in size; big size came from earlier responsive explorations.
- got them centering, though at the expense of fixing the width of that set of options.  Unfortunately, CSS doesn't [let you](https://developer.mozilla.org/en-US/docs/Web/CSS/width) have the width of a parent and a child depend on each other, which is what I really wanted: a child that would maximise its width unless constrained by a parent, but also a parent that would constrain its width to that of its child so that it could be then centered.  Fortunately, most viewers will probably see this at desktop width, in which case the hardcoded narrow width should be fine.

**Comment by tanyaparker** [comment]
Screenshots LGTM - thanks!

**Pull request merged**